user-service/src/main/java/rs/edu/raf/banka1/controller/UserController.java

```diff
@@ -159,6 +154,32 @@ public ResponseEntity<ActivateAccountResponse> activateAccount(@PathVariable Str
         return new ResponseEntity<>(userService.activateAccount(token, password), HttpStatus.OK);
     }
 
+    @PostMapping(value = "/reset/{email}")
+    @Operation(summary = "Reset password", description = "Send password reset email to user")
+    @ApiResponses({
+            @ApiResponse(responseCode = "200", description = "Email with password reset URL sent successfully",
+                    content = {@Content(mediaType = "application/json",
+                        schema = @Schema(implementation = ActivateAccountResponse.class))}),
+            @ApiResponse(responseCode = "500", description = "Internal server error")
+    })
+    public ResponseEntity<Boolean> sendResetPasswordEmail(@PathVariable String email) {
+        return new ResponseEntity<>(userService.sendResetPasswordEmail(email), HttpStatus.OK);
```

Comment:
This shouldn't return true or false for the same reason that it shouldn't return 404. Just 200 is fine